src/main/java/com/example/onlineshop_frontend/dto/UserDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.example.onlineshop_frontend.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UserDTO {
+    private Long id;
+    private String firstName;
+    private String lastName;
+    private String username;
+    private String email;
+    private String password;
```

Comment:
we need a 'repeatPassword' field here (i assume this is the DTO for User registration)

Reply:
Agree. Or in case it's for response - we'll need to remove the password from it.